src/screens/confirm-password/confirm-password.jsx

```diff
@@ -0,0 +1,39 @@
+import Btn from '../../components/btn/btn';
+import GameTitle from '../../components/game-title/game-title';
+import Input from '../../components/Input/Input';
+import ScreenWrapper from '../../components/wrappers/screen-wrapper/screen-wrapper';
+import { useNavigate } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+
+function ConfirmPassword() {
+  const navigate = useNavigate();
+
+  return (
+    <ScreenWrapper>
+      <GameTitle />
+      <form className="form-wrapper">
+        <Input id={uuidv4()} placeholder="Password" type="password" />
+        <Input id={uuidv4()} placeholder="Password" type="password" />
```

Comment:
`name` instead of `id`

don't use `uuid` for name, give for inputs appropriate name

Reply:
apply this for all your form inputs